newsletters/templates/newsletters/create.html

```diff
@@ -0,0 +1,25 @@
+{% extends 'tgrsite/main.html' %}
+{% block title %}
+New Newsletter
+{% endblock %}
+{% block body %}
+
+<h1>New Newsletter</h1>
+<form action="" method="POST">
```

Comment:
Is it okay for `action` to be blank

Reply:
All of the examples on Django's documentation have blank actions [(see here)](https://docs.djangoproject.com/en/2.1/ref/class-based-views/generic-editing/#django.views.generic.edit.CreateView) so I think it's fine? Will try removing the action altogether to see if it still works.

Reply:
Fixed.